lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -89,7 +94,9 @@ def test_proctoring_tab_visible_for_course_staff_and_admin(self, enable_proctore
         # verify that proctoring tab is visible for course instructor
         CourseStaffRole(self.course.id).remove_users(self.instructor)
         CourseInstructorRole(self.course.id).add_users(self.instructor)
-        self._assert_proctoring_tab_is_available()
+        response = self._assert_proctoring_tab_is_available()
+        # the default backend does not support the review dashboard
+        self.assertNotIn('Review Dashboard', response.content)
```

Comment:
Nit: I'd put this in the `test_review_dashboard` function if it's not too much work. I was kind of confused why it was in here, since `test_proctoring_tab_visible_for_course_staff_and_admin` doesn't seem to be responsible for testing that.